### PR TITLE
Update js-fun-at-a-restaurant's extension to no longer include classes

### LINKED ISSUE #577: Update first solo project 1
# Project to be updated: [JS Fun at a restaurant](https://frontend.turing.edu/projects/module-1/restaurant.html)

## Summary of changes needed: 
- This project currently asks students to implement a Class (Chef) based on a test suite
- The project needs to be updated so the the tests are not expecting a Class to be created
- The boilerplate should be removed from the Chef.js file
- Chef is an extension for the project, so a different extension should replace it

### PR DESCRIPTION
**Pull request by Kalikoze**
### Description of Changes
I've updated the project spec to no longer mention anything about classes for the extension.  Instead, the extension has students looking over the **Dot and Bracket Notation* lesson and refactoring some of the functions to utilize bracket notation.  

### Commentary
For some background, I initially had started adding additional tests for students to work through for the extension.  After discussing further with @kaylagordon, we discussed that it didn't make much sense to get more complex for the extension since the tests in the `restaurant-test.js` file are already quite challenging for them in week 1, and we thought it might be more valuable for students to practice refactoring and DRYing up their code as an extension.  

Some tests were moved from the `Chef-test.js` into the `restaurant-test.js` file but have been refactored slightly so they no longer use classes.  You can see more in the [js-fun-at-a-restaurant PR](https://github.com/turingschool-examples/js-fun-at-a-restaurant/pull/28)

### Questions for reviewers
* Does the extension still feel sufficient?

Closes #577

### CODE REVIEW

File: projects/module-1/restaurant.md [hunk 1]
```diff
@@ -76,11 +76,11 @@ problem.
     solutions. Is this the best approach to solving the problem? Is there a
     different way you could make the tests pass?  
 
-### Extension (not part of the project, just stuff to do if you get to it): Complete the Chef tests
-  - Unskip the first test in `test/Chef-test.js`, and get to work passing the tests
-  - Run `npm test test/Chef-test.js`
-  - This test involves using a class. It will require self-teaching in order to complete.
-  - Ensure that all of the skips are removed from the test file when you push up to GitHub.
+### Extension: Refactoring
```
Comment (CassandraGoose) [inline projects/module-1/restaurant.md:79]: I particularly like the idea of making refactoring an extension for this level and the level of challenge that you created here. It seems achievable for some students and it gets them thinking more deeply about the code they are writing - it moves their brain from purely thinking about "making things just work" to "what makes good software".

**Review by CassandraGoose — APPROVED** [review]
I read through [this PR ](https://github.com/turingschool-examples/js-fun-at-a-restaurant/pull/28/files)to confirm that checkForFood doesn't match up to a class-based challenge anymore. 🎉 

Looks good to me.

**Comment by nicktu12** [comment]
Nice 😎 Looks good to me

**Comment by Kalikoze** [comment]
Thanks for the feedback Cass and Nick!  Merging into the development branch!

**Pull request merged**